cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -118,6 +119,11 @@ func (m command) build() *cli.Command {
 				Usage:       "path to the config file for the target runtime",
 				Destination: &config.configFilePath,
 			},
+			&cli.StringFlag{
+				Name:        "nvidia-config",
```

Comment:
Looking at this again, I don't think `nvidia-config` is specific enough. Something like `drop-in-config` may be better at communicating the intent here.